msgpack-core/src/test/scala/org/msgpack/core/MessagePackerTest.scala

```diff
@@ -235,6 +236,25 @@ class MessagePackerTest
       up1.hasNext shouldBe false
       up1.close
     }
+
+    "pack a lot of String within expected time" in {
+      val count = 500000
+
+      def measureDuration(outputStream: java.io.OutputStream) : Long = {
+        val packer = MessagePack.newDefaultPacker(outputStream)
+        val start = Platform.currentTime
+        for (i <- 0 to count) {
+          packer.packString("0123456789ABCDEF")
+        }
+        packer.close
+        Platform.currentTime - start
+      }
+
+      val baseDuration = measureDuration(new ByteArrayOutputStream())
+      val (_, fileOutput) = createTempFileWithOutputStream
+      val targetDuration = measureDuration(fileOutput)
+      targetDuration shouldBe <= (baseDuration * 3)
```

Comment:
The performance of these two blocks can be affected by the execution order because of JVM warm-up, JIT compilation, etc. And that is why I used `time` and `block` for performance comparison: https://github.com/msgpack/msgpack-java/blob/v07-develop/msgpack-core/src/test/scala/org/msgpack/core/MessageUnpackerTest.scala#L309

For example, 
```
val t = time("bench", repeat=10) {
  block("A") { ... }
  blcok("B") { ... }
}
```
It executes A, B, A, B, A, B, ...  You can reference the execution time average without min/max values with `t("A").averageWithoutMinMax`